packages/app/src/cli/validators/extensions/ui.ts

```diff
@@ -1,26 +1,18 @@
 import {UIExtension} from '../../models/app/extensions.js'
+import {isValidUrl} from '../../utilities/extensions/url.js'
 import {error} from '@shopify/cli-kit'
 
-const WebPixelConfigError = (property: string) => {
-  return new error.Abort(
-    `The Web Pixel Extension configuration is missing the key "${property}"`,
-    `Please update your shopify.ui.extension.toml to include a valid "${property}"`,
-  )
-}
-
-const DeprecatedPropertyError = (deprecatedProperty: string, replacementProperty?: string) => {
-  return new error.Abort(
-    `The property "${deprecatedProperty}" is deprecated and no longer supported.`,
-    replacementProperty ? `It has been replaced by "${replacementProperty}".` : '',
-  )
-}
-
 export function validateUIExtensions(extensions: UIExtension[]) {
   for (const extension of extensions) {
     switch (extension.type) {
       case 'web_pixel_extension':
         validateWebPixelConfig(extension)
         break
+
+      case 'customer_accounts_ui_extension':
+        validateAuthenticatedUrls(extension)
```

Comment:
I suppose that at some point those extension specific validators will go into their own files, but we can keep it like this for now.